File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/CreateProcessingJobPackagesRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class CreateProcessingJobPackagesRequest {
    @JsonProperty("ExpirationDate")
    public String expirationDate;
    @JsonProperty("FinishDate")
    public String finishDate;
    @JsonProperty("FinishNote")
    public String finishNote;
    @JsonProperty("FinishProcessingJob")
    public Boolean finishProcessingJob;
    @JsonProperty("Item")
    public String item;
    @JsonProperty("JobName")
    public String jobName;
    @JsonProperty("Location")
    public String location;
    @JsonProperty("Note")
    public String note;
    @JsonProperty("PackageDate")
    public String packageDate;
    @JsonProperty("PatientLicenseNumber")
    public String patientLicenseNumber;
    @JsonProperty("ProductionBatchNumber")
    public String productionBatchNumber;
    @JsonProperty("Quantity")
    public Integer quantity;
    @JsonProperty("SellByDate")
    public String sellByDate;
    @JsonProperty("Sublocation")
    public String sublocation;
    @JsonProperty("Tag")
    public String tag;
    @JsonProperty("UnitOfMeasure")
    public String unitOfMeasure;
    @JsonProperty("UseByDate")
    public String useByDate;
    @JsonProperty("WasteCountQuantity")
    public String wasteCountQuantity;
    @JsonProperty("WasteCountUnitOfMeasureName")
    public String wasteCountUnitOfMeasureName;
    @JsonProperty("WasteVolumeQuantity")
    public String wasteVolumeQuantity;
    @JsonProperty("WasteVolumeUnitOfMeasureName")
    public String wasteVolumeUnitOfMeasureName;
    @JsonProperty("WasteWeightQuantity")
    public String wasteWeightQuantity;
    @JsonProperty("WasteWeightUnitOfMeasureName")
    public String wasteWeightUnitOfMeasureName;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/CreateProcessingJobPackagesRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class CreateProcessingJobPackagesRequest(
    @JsonProperty("ExpirationDate")
    val expirationDate: String? = null,
    @JsonProperty("FinishDate")
    val finishDate: String? = null,
    @JsonProperty("FinishNote")
    val finishNote: String? = null,
    @JsonProperty("FinishProcessingJob")
    val finishProcessingJob: Boolean? = null,
    @JsonProperty("Item")
    val item: String? = null,
    @JsonProperty("JobName")
    val jobName: String? = null,
    @JsonProperty("Location")
    val location: String? = null,
    @JsonProperty("Note")
    val note: String? = null,
    @JsonProperty("PackageDate")
    val packageDate: String? = null,
    @JsonProperty("PatientLicenseNumber")
    val patientLicenseNumber: String? = null,
    @JsonProperty("ProductionBatchNumber")
    val productionBatchNumber: String? = null,
    @JsonProperty("Quantity")
    val quantity: Int? = null,
    @JsonProperty("SellByDate")
    val sellByDate: String? = null,
    @JsonProperty("Sublocation")
    val sublocation: String? = null,
    @JsonProperty("Tag")
    val tag: String? = null,
    @JsonProperty("UnitOfMeasure")
    val unitOfMeasure: String? = null,
    @JsonProperty("UseByDate")
    val useByDate: String? = null,
    @JsonProperty("WasteCountQuantity")
    val wasteCountQuantity: String? = null,
    @JsonProperty("WasteCountUnitOfMeasureName")
    val wasteCountUnitOfMeasureName: String? = null,
    @JsonProperty("WasteVolumeQuantity")
    val wasteVolumeQuantity: String? = null,
    @JsonProperty("WasteVolumeUnitOfMeasureName")
    val wasteVolumeUnitOfMeasureName: String? = null,
    @JsonProperty("WasteWeightQuantity")
    val wasteWeightQuantity: String? = null,
    @JsonProperty("WasteWeightUnitOfMeasureName")
    val wasteWeightUnitOfMeasureName: String? = null
)
